backend/routes/api/resources.js

```diff
@@ -91,6 +92,31 @@ router.get(
   })
 );
 
+// get list of resources filtered by tag options
+router.get(
+  "/filter/tags",
```

Comment:
Is there a possibility that we can add this functionality into `/filter` instead? From my understanding of the issue, I think people would be entering tags to the search in addition to the location and name searching, not solely filtering by tags.

Reply:
@angadgarg25 The reason I didn't add it into `/filter` is because that is used in the Directory view for a different purpose. It uses a fuzzy search to find matches that are close to the search field. However, for this route, I only wanted to filter by direct results. So after searching for location/name, if a suggested tag pops up, you could click on it, and it would then filter the results you already have.

Reply:
So yeah we would also need another route to suggest tags based on the input search field as well.

Reply:
oh i see that makes sense. so then we need another route that's almost the same as `/filter` but it takes multiple tags and does an exact match rather than doing fuzzy search on a single tag correct? I'll start working on the plain UI of adding and removing tags to the search on the map page first and whoever can pick up the backend change.